x-pack/plugins/fleet/common/types/models/package_spec.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { RegistryPolicyTemplate } from './epm';
+
+// Based on https://github.com/elastic/package-spec/blob/master/versions/1/manifest.spec.yml#L8
+export interface PackageSpecManifest {
```

Comment:
Need to correct these some are listed as required but are not e.g `type` https://github.com/elastic/kibana/issues/83426